handbook/openid-connect/openid-connect.md

```diff
@@ -338,3 +367,22 @@ To resolve this issue:
 3. Modify the number at the end to a number larger than X, where X is the required size of the cache entry specified in the error log.
 
 4. Restart the NI Web Server from the `Control` tab of the NI Web Server Configuration application.
+
+### ID Token Management Encryption
+
+If the provider is configured to use an asymmmetric ID token management encryption algorithm and private keys are missing or incorrect, or if the provider is configured to use an unsupported ID token management encryption algorithm, the user will be redirected back to the SystemLink login page after logging in to the provider.
+
+!!! note "Example error logs"
+    When this happens, the NI Web Server error logs will contain entries containing the following:
```

Comment:
> When this happens,

Opportunity to clarify whether it's the act of redirecting the user that logs the error (which is what it sounds like now), or something else